Parsers/ASimAuthentication/Parsers/vimAuthenticationCiscoMerakiSyslog.yaml

```diff
@@ -0,0 +1,232 @@
+Parser:
+  Title: ASIM Authentication parser for Cisco Meraki
+  Version: '0.2.1'
+  LastUpdated: June 7 2024
```

Comment:
"LastUpdated" property exists but is not correct format. The expected format is, for example, "Jun 29, 2024"

Reply:
Done